pkg/util/perf/perf_linux.go

```diff
@@ -30,14 +31,17 @@ import (
 )
 
 type PerfCollector struct {
+	CgroupFile        *os.File
```

Comment:
How about encapsulating a method like `(*PerfCollector).Cleanup()`, not exposing the detailed behavior like closing a cgroup fd.

Reply:
Good idea! Already modified.